integration/test_integration_basics.py

```diff
@@ -207,3 +207,14 @@ def test_h2_tls_context_configuration(self):
     ])
     counters = self.getNighthawkCounterMapFromJson(parsed_json)
     self.assertEqual(counters["ssl.ciphers.ECDHE-RSA-AES256-GCM-SHA384"], 1)
+
+  def test_prefetching(self):
+    """
+    Test we prefetch connections.
+    """
+    parsed_json = self.runNighthawkClient([
+        "--duration 1", "--rps 1", "--prefetch-connections", "--connections 50",
+        self.getTestServerRootUri()
+    ])
+    counters = self.getNighthawkCounterMapFromJson(parsed_json)
+    self.assertEqual(counters["upstream_cx_http1_total"], 50)
```

Comment:
How does this verify that connections were prefetched?

Reply:
Added a comment to clarify; without prefetching we'd expect a single connection because we
test at 1 rps for 1 second.